Library/Formula/spacenavd.rb

```diff
@@ -0,0 +1,47 @@
+class Spacenavd < Formula
+  desc "Userspace device driver/daemon for 3Dconnexion's 3D input devices."
+  homepage "http://spacenav.sourceforge.net/index.html"
+  url "https://downloads.sourceforge.net/project/spacenav/spacenav%20daemon/spacenavd%200.6/spacenavd-0.6.tar.gz"
+  sha256 "c2d203bf96c5a959590146a43fe5d6e5e8c5c38a8b2f55aa199d967d0d88d0ab"
+
+  option "with-x11", "Enable support for x11 communication."
+
+  depends_on :x11 => :optional
+
+  patch :DATA
+
+  def install
+    args = %W[
+      --disable-debug
+      --disable-dependency-tracking
+      --disable-silent-rules
+      --prefix=#{prefix}
+    ]
+
+    args << "--disable-x11" if build.without? "x11"
+
+    system "./configure", *args
+
+    system "make", "install"
+  end
+
+  test do
+    system "spacenavd", "-h"
```

Comment:
If it's placing things in `/var` it should be set so it uses `HOMEBREW_PREFIX/"var"` instead as per our standard.

If there's a daemon you could do something like [this](https://github.com/Homebrew/homebrew/blob/master/Library/Formula/mpd.rb#L127-L138) but `test do` blocks aren't supposed to create or write any file outside of the `testpath`, so we'd need to redirect the `pid` creation if possible.